libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -476,7 +476,7 @@ bool AC_WPNav::advance_wp_target_along_track(float dt)
 
     float vel_time_scalar = 1.0;
     if (is_positive(_wp_desired_speed_xy_cms)) {
-        update_vel_accel(_terrain_vel, _terrain_accel, dt, false);
```

Comment:
This looks like a good fix but just for the record what was there was clearly a typo.  Luckily the "false" would have been cast to a float so no impact.

Reply:
Yeh, exactly.